TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robot/subsystems/realsenseloader/RealsenseManager.java

```diff
@@ -46,7 +47,11 @@ public static void init(HardwareMap hardwareMap) {
         if (slamera == null) {
             slamera = new T265Camera(cameraRobotOffset, encoderMeasurementCovariance, hardwareMap.appContext);
         }
-        slamera.setPose(new com.arcrobotics.ftclib.geometry.Pose2d());
+        slamera.setPose(PoseUtil.toFtclibPose(
```

Comment:
I'm doing it like this now so that op modes can set this init position rather than using the (possibly flaky) `robot.drivetrain.setPoseEstimate()`